fiftyone/factory/repos/delegated_operation.py

```
@@ -613,13 +572,10 @@ def count(self, filters: dict = None, search: dict = None) -> int:
 
         return self._collection.count_documents(filter=query)
 
-    def ping(self, _id: ObjectId, log_tail: str = None):
-        updates = {"updated_at": datetime.utcnow()}
-        if log_tail and isinstance(log_tail, str):
-            updates["log_tail"] = log_tail
+    def ping(self, _id: ObjectId):
         self._collection.update_one(
             filter={"_id": _id},
-            update={"$set": updates},
+            update={"$set": {"updated_at": datetime.utcnow()}},
```

Ruff (0.15.12)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)